gen_wrap.py

```diff
@@ -1368,14 +1368,18 @@ def emit_context_check(arg_idx, arg_name):
 
             docs.append(":param %s: string" % arg.name)
 
-        elif arg.base_type == "int" and arg.ptr == "*":
+        elif arg.base_type == "isl_bool" and arg.ptr == "*":
```

Comment:
Please add an assertion that supports your claim that no more `int *` arguments are present.

Reply:
I changed it to support both isl_bool and int cases. I think it is better in terms of backward compatibility.